README.md

```diff
@@ -1,11 +1,10 @@
-Terraform Provider
+Terraform Provider for Google Cloud Platform
 ==================
 
 - Website: https://www.terraform.io
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
-![Terraform](https://rawgithub.com/hashicorp/terraform/master/website/source/assets/images/logo-hashicorp.svg)
```

Comment:
What happens if we try the new location of the file? https://rawgit.com/hashicorp/terraform-website/master/source/assets/images/logo-hashicorp.svg